ccd-definition/FixedLists.json

```diff
@@ -122,109 +122,109 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "LivingSituationList",
-    "ListElementCode": "Voluntarily in section 20 care order",
+    "ListElementCode": "VOLUNTARILY_IN_SECTION_20_CARE_ORDER",
     "ListElement": "Voluntarily in section 20 care order"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "LivingSituationList",
-    "ListElementCode": "In hospital and soon to be discharged",
+    "ListElementCode": "IN_HOSPITAL_AND_SOON_TO_BE_DISCHARGED",
     "ListElement": "In hospital and soon to be discharged"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "LivingSituationList",
-    "ListElementCode": "Other",
+    "ListElementCode": "OTHER",
     "ListElement": "Other"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "FathersResponsibilityList",
-    "ListElementCode": "Yes",
+    "ListElementCode": "YES",
```

Comment:
There is another YES / NO / I DON'T KNOW question for litigation capcity issues. This can be refactored into a generic yes no don't know field